README.md

```diff
@@ -2,7 +2,10 @@
 
 [![Build Status](https://travis-ci.org/GoogleCloudPlatform/nodejs-docker.svg?branch=master)](https://travis-ci.org/GoogleCloudPlatform/nodejs-docker)
 
-This repository contains the source for the `gcr.io/google_appengine/nodejs` [docker](https://docker.com) image. This image can be used as the base image for running applications on [Google App Engine Managed VMs](https://cloud.google.com/appengine), [Google Container Engine](https://cloud.google.com/container-engine), or any other Docker host.
+This repository contains the source for the Google-maintained Node.js [docker](https://docker.com) image.
+This image can be found at `launcher.gcr.io/google/nodejs` or `gcr.io/google-appengine/nodejs`, and this image can be used as the base image
```

Comment:
are we just using `launcher.gcr.io/google...` and `l.gcr.io/google...` interchangeably?